### ISSUE TITLE
Canvas - default size (ECOMP req)

### ISSUE THREAD

**Issue by paulinshek** [att/rcloud.rcap#95]
Currently, the canvas sizes to the window of the browser. We need an option to have it size to a smaller scope.
-Based on tag it is being opened in (e.g., iFrame, ??)
-Specific size enter by User or entered on URL: (Guy)
Having RCAP detect the size of it’s window and adjust automatically would be very useful. It would be even better if we could explicitly specify width and height parameters in the URL when we call an RCAP notebook, which would override anything else.
   --So, for example, <RCAP NOTEBOOK URL>?width=400+height=300.

**Comment by gaborcsardi** [comment]
If possible, make the sizes available for the notebook. This could be easy with the `rcloud.params` package.

**Comment by shaneporter** [comment]
The canvas sizes with the following code:

```javascript
$('#inner-stage').css({
    'width' : (screen.width - 20).toString() + 'px',
    'margin-left' : 'auto',
    'margin-right' : 'auto'
});
```

So, it centres it, and subtracts 20 from the available screen width.

**Comment by shaneporter** [comment]
The width can be changed quite easily. This contracts all controls' widths accordingly.

The height for each row is fixed. Therefore, reducing the height of the canvas does not change the height of the controls.

**Comment by shaneporter** [comment]
This was implemented in PR #134, some documentation is available at https://github.com/att/rcloud.rcap/wiki/Embedding-RCAP-in-an-iframe.
